.azure/pipelines/ci.yml

```diff
@@ -577,9 +577,20 @@ stages:
       agentOs: Windows
       timeoutInMinutes: 180
       steps:
-      - script: .\restore.cmd -ci
-        displayName: Restore
-      - script: .\build.cmd -ci -NoRestore -test -projects eng\helix\helix.proj /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildAllProjects=true /p:BuildNative=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log -bl
+      # Build the x86 shared framework
+      # TODO: make it possible to build for one Windows architecture at a time
+      # This is going to actually build x86 native assets. See https://github.com/aspnet/AspNetCore/issues/7196
```

Comment:
Please remove all references to #7196. That bug is closed because it just doesn't matter. The first sentence here is however correct.